OWNERS_ALIASES

```diff
@@ -146,8 +143,10 @@ aliases:
     - foxish
     - liyinan926
   ug-vmware-users-leads:
+    - brysonshepherd
     - cantbewong
     - mylesagray
+    - phenixblue
```

Comment:
Q: are TechnicalLeads eligible for being in "-leads"?
this is the result of using the new Owners() method consistently across the template.

Reply:
Yes, this is fine.